cosmic-client/src/main/webapp/scripts/instances.js

```diff
@@ -2825,7 +2829,7 @@
             allowedActions.push("stop");
             allowedActions.push("restart");
 
-            if ((jsonObj.hypervisor != 'KVM' || g_kvmsnapshotenabled == true)) {
+            if (jsonObj.hypervisor != 'LXC') {
```

Comment:
@eliasgomes fyi LXC has been removed from Cosmic so you can remove all references to it